src/containers/Form/index.js

```diff
@@ -127,21 +130,34 @@ class Form extends Component {
         </FormGroup>
         <FormGroup title='Prestataire'>
           <TextInput
+            ref={'prestataire'}
             value={this.props.recipient}
             placeholder='Fournisseur ou prestataire...'
+            returnKeyType={'next'}
+            autoCorrect={false}
+            onSubmitEditing={(event) => { 
+              this.refs.description.focus(); 
+            }}
             onChangeText={(recipient) => this.props.updateRecipient(recipient)}
             style={styles.textInput}
           />
         </FormGroup>
         <FormGroup title='Description'>
           <TextInput
+            ref='description'
             value={this.props.description}
+            autoCorrect={false}
             placeholder='Description...'
+            onSubmitEditing={(event) => { 
+              this.refs.amount.focus(); 
```

Comment:
I have an error : 
<img width="200" alt="screenshot" src="https://user-images.githubusercontent.com/12433625/28747932-b3185566-74a2-11e7-84d6-51a9f5757a5f.png">

Maybe you can pass the TextInput reference of `AmountConverter` component to parent component (the `Form`). See: https://facebook.github.io/react/docs/refs-and-the-dom.html#exposing-dom-refs-to-parent-components